modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/add_to_cart/AddToCartButton.js

```diff
@@ -126,7 +126,7 @@ function AddToCartButton({
 					notAllowed ||
 					(cartAtomState.updating && !isTriggeringCartUpdate),
 			})}
-			disabled={buttonDisabled}
+			disabled={buttonDisabled || cartAtomState.updating}
```

Comment:
Hey @brianikim, this may temporarily solve the issue, but for the LPP from which it stems, the issue is broader. Since the customer is on 2024.q1 let's talk offline, back then there was still the huge performance bottleneck for the big PATCH on add to cart interaction. See: c98a0d87b93f0d38a5754c47e85b9b24f4e1e21b for instance, which we could evaluate to backport.